### DISCUSSION TITLE
guide/get-started

### DISCUSSION THREAD

**Discussion by giscus** [Mereithhh/vanblog-comment#15] [Announcements]
# guide/get-started

欢迎使用 VanBlog ，只需几个步骤，你就可以在你的服务器搭建自己的博客服务了。 目前 VanBlog 还在快速迭代中，如果后台出现升级提示，推荐进行升级。 介绍 include(@/info.snippet.md) 配置要求 理论上 VanBlog 不需要很高的配置，实际上演示站不算数据库，资源的占用情况如图： 资源占用 不到 400M 的内存（...

https://vanblog.mereith.com/guide/get-started.html

**Comment by yulongsa** [comment]
宝塔不能一键部署呀，要是能一键部署就好了

**Comment by CnDaiL** [comment]
请问，支持虚拟主机吗

**Reply by owenzjg** [reply]
我问过作者了，不支持

**Comment by yxd129220** [comment]
目前v0.54.0版本的arm64 docker部署了之后无法初始化，直接就是登录界面，查看log目录也没有restore.key文件

**Reply by yxd129220** [reply]
嗯，是正常的，之前是内存不够

**Comment by MemoryCollection** [comment]
建议加一个1panel的部署方法 很简单的

**Comment by Qing1** [comment]
本人的宝塔面板一键部署，以及相应的Nginx反代教程说明，供大家参考：https://daohang.bxmang.top/post/3

**Reply by chuilishi** [reply]
文章怎么不见了😂
我这边反代配置了但是没跑起来

**Reply by 13969526885** [reply]
启动时总是显示 vanblog Error Get "https://registry-1.docker.io/v2/": net/http: request canceled while waiting for connection (Client.Timeout...               15.0s 
 ✘ mongo Error   context canceled                                                                                                                 15.0s 
Error response from daemon: Get "https://registry-1.docker.io/v2/": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
启动失败，请稍后查看日志信息，请问咋回事啊？

**Reply by 13969526885** [reply]
在宝塔面板一键部署也是在"https://registry-1.docker.io/v2/":这里出错

**Comment by 13969526885** [comment]
感谢作者的贡献，能否指导一下服务器部署，阿里云服务器，宝塔面板，无论是一键部署还是docker部署，都在Get "https://registry-1.docker.io/v2/": 时出错，这是什么原因啊?我也尝试了下载压缩包在宝塔安装，但是里面没有首页文件，index.html，网站打开了是空白的

**Reply by 13969526885** [reply]
脚本部署时还出现 WARN[0000] /var/vanblog/docker-compose.yaml: `version` is obsolete

**Reply by Myhuiku** [reply]
你更换一下阿里云的镜像，换成docker的，然后直接用脚本，我的这样好了

**Comment by AlienC** [comment]
mongo能换成mysql吗？

**Comment by hcieget110** [comment]
Error response from daemon: Get "https://registry-1.docker.io/v2/": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
您好，请问这个报错怎么解决
